src/app/components/HoldConfirmation/HoldConfirmation.jsx

```diff
@@ -165,6 +165,55 @@ class HoldConfirmation extends React.Component {
     const pickupLocation = this.props.location.query.pickupLocation;
     let deliveryLocation = null;
 
+    const item = (!this.props.location.query.errorStatus && !this.props.location.query.errorMessage)
```

Comment:
I don't think `item` works as a variable name here. It's the confirmation page so it probably makes more sense if it's `confirmationInfo` rather than item.